packages/clis/generator/index.ts

```diff
@@ -591,8 +604,19 @@ function handleCitiesCommand(args: ReturnType<typeof citiesCommandBuilder>) {
   const toJsonPath = (map: 'usa' | 'europe', suffix: string) =>
     path.join(args.inputDir, `${map}-${suffix}.json`);
 
-  const cityAndCountryFeatures = args.map.flatMap(map => {
-    const countries = readArrayFile<Country>(toJsonPath(map, 'countries'));
+  const countryIsoA2 = createIsoA2Map();
+  const cityAndCountryFeatures = [args.map].flat().flatMap(map => {
+    const countries = readArrayFile<Country>(toJsonPath(map, 'countries')).map(
+      c => {
+        if (map === 'europe') {
+          return {
+            ...c,
+            code: countryIsoA2.get(c.code),
+          };
```

Comment:
Going through these hoops of converting in-game country codes to [ISO 3166-1 Alpha 2 codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2), because [I had originally requested](https://forum.scssoft.com/viewtopic.php?p=1902372#p1902372) that they be done that way... for reasons I can't remember 😕.